packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -154,6 +149,231 @@ export interface NftDetectionConfig extends BaseConfig {
   selectedAddress: string;
 }
 
+export type ReservoirResponse = {
+  tokens: TokensResponse[];
+  continuation?: string;
+};
+
+export type TokensResponse = {
+  token: TokenResponse;
+  ownership: Ownership;
+  market?: Market;
+  blockaidResult?: Blockaid;
+};
+
+export enum BlockaidResultType {
+  Benign = 'Benign',
+  Spam = 'Spam',
+  Warning = 'Warning',
+  Malicious = 'Malicious',
+}
+
+export type Blockaid = {
+  contract: string;
+  chainId: number;
+  result_type: BlockaidResultType;
+  malicious_score: string;
+  attack_types: object;
+};
+
+export type Market = {
+  floorAsk?: FloorAsk;
+  topBid?: TopBid;
+};
+
+export type TokenResponse = {
+  chainId: number;
+  contract: string;
+  tokenId: string;
+  /** @description Can be erc721, erc115, etc. */
```

Comment:
Do you want this description to show when the property is hovered over? If so, the `@description` isn't necessary here — if any property is preceded by a `/** ... */` comment, then it automatically becomes the description. I gave an example of this below but you'd want to apply this to the other ones below.

That said, some of these JSDoc annotations seem out of place:

- Properties don't have defaults, so the `@default` tag doesn't seem right here. Perhaps you want to note the default in whichever function receives an argument of this type?
- The comment for `lastAppraisalValue` says that it can be null, but that's not what the type says. Why? Perhaps there is another place where `lastAppraisalValue` can be null?

Reply:
Thnx @mcmire  for catching those, i ve copied those types from their official SDK, with just a few tweaks, 
I will cleanup all those descriptions and defaults 🙏